native/SampleApps/RestAPIExplorer/RestAPIExplorer/RestAPIExplorer-Info.plist

```diff
@@ -29,7 +40,7 @@
 	<key>NSMainNibFile~ipad</key>
 	<string></string>
 	<key>SFDCOAuthLoginHost</key>
-	<string>login.salesforce.com</string>
+	<string>test.salesforce.com</string>
```

Comment:
I can revert that line but I always use it against sandbox and I got tired of switching hosts.